acceptance/tests/resource/exec/should_run_command_in_cwd.rb

```diff
@@ -1,6 +1,7 @@
 test_name "The Exec resource should run commands in the specified cwd" do
   tag 'audit:high',
       'audit:acceptance'
+  confine :except, :platform => /debian-12-amd64/ # PUP-5358
```

Comment:
```suggestion
  confine :except, :platform => /debian-12-amd64/ # PUP-12020
```

Reply:
this is why we can't have nice things :(